sn_client/src/wallet.rs

```diff
@@ -152,9 +152,9 @@ impl WalletClient {
             // just skip it as it will then get verification failure,
             // and repay/re-upload will be triggered correspondently.
             match res {
-                Ok((content_addr, Ok(cost))) => {
```

Comment:
this _is_ plural no? ie, a vec of the various `(mainkey, token)`s required to fulfill payment?